### ISSUE TITLE
votesvalid, votesinvalid, votescast are numbers not strings

### ISSUE THREAD

**Issue by GabrielInTheWorld** [OpenSlides/openslides-backend#488]
I tried to create a new poll with the following data:
```
[{'action': 'poll.create', 'data': [{'meeting_id': 1, 'title': 'Abstimmung', 'pollmethod': 'YNA', 'publish_immediately': True, 'type': 'analog', 'options': [{'content_object_id': 'motion/4', 'Y': 2, 'A': 4, 'N': 20}], 'votescast': 80, 'votesinvalid': 4, 'votesvalid': 22, 'content_object_id': 'motion/4'}]}]
```

The backend sends me the following error:
`Fehler: data.votesvalid must be string or null`

But, votesvalid, votesinvalid and votescast has to be numbers as it is defined the [openslides-backend-coordination](https://github.com/OpenSlides/openslides-backend-coordination/issues/117). Am I right?

**Comment by FinnStutzenstein** [comment]
Right, they must be converted to a decimal representation internally.

**Comment by FinnStutzenstein** [comment]
Quick question: Is the client allowed to enter fractional values like `12,54`?

**Comment by GabrielInTheWorld** [comment]
> Quick question: Is the client allowed to enter fractional values like `12,54`?

Entering is allowed, but the client sends only an integer.

**Comment by FinnStutzenstein** [comment]
We should speak about it.

**Comment by FinnStutzenstein** [comment]
As discussed: The client should send decimal numbers.
